smarts/core/tests/test_notebook.py

```diff
@@ -26,7 +26,9 @@
 import gym
 import importlib_resources
 import pytest
+# pytype: disable=import-error
 import pytest_notebook.nb_regression as nb
+# pytype: enable=import-error
```

Comment:
I think this can be removed by `pip install smarts[test]` in the type test GitHub action.

Reply:
1. No need for individual `pytype: disable=import-error` commands anymore as we have ignored `import-error` globally during pytest in Github CI.
https://github.com/huawei-noah/SMARTS/blob/3ae9f8479d5ebf81f9a79e52d2eb94332f6c7acd/.github/workflows/ci-format.yml#L72  
2. Thus, I think no need to include `test` option while installing smarts in Github CI.